pallets/stateful-storage/src/lib.rs

```diff
@@ -178,6 +178,10 @@ pub mod pallet {
 		PaginatedPageRemoved { msa_id: MessageSourceId, schema_id: SchemaId, page_id: PageId },
 	}
 
+	type ItemizedFullKey = (SchemaId,);
```

Comment:
Can we please add some docs for this type,

Reply:
Removed; after merge from pg-main, got `ItemizedKey` from `common_primitives::stateful_storage`